opal/datatype/opal_datatype_module.c

```diff
@@ -204,6 +207,21 @@ int opal_datatype_register_params(void)
     return OPAL_SUCCESS;
 }
 
+static void opal_datatype_finalize (void)
+{
+    /* As the synonyms are just copies of the internal data we should not free them.
+     * Anyway they are over the limit of OPAL_DATATYPE_MAX_PREDEFINED so they will never get freed.
+     */
+
```

Comment:
Let’s invert these 2 so that the destruction of the master convertors could use the dedicated opal_datatype_dfd.

Reply:
Ok, makes sense. Will make that change.

Reply:
Fixed.

Reply:
👍